scripts/verify_week2_migration.py

```
@@ -0,0 +1,127 @@
+    print("=" * 40)
+    
+    conn = sqlite3.connect("data/dealgenie.db")
+    cursor = conn.cursor()
+    
+    # Get all table names
+    cursor.execute("SELECT name FROM sqlite_master WHERE type='table' ORDER BY name")
+    tables = [row[0] for row in cursor.fetchall()]
+    
+    for table in tables:
+        try:
+            cursor.execute(f"SELECT COUNT(*) FROM {table}")
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)